test/e2e/proxy.go

```diff
@@ -155,9 +155,6 @@ func proxyContext(version string) {
 					if e, a := val, string(body); e != a {
 						recordError(fmt.Sprintf("%v: path %v: wanted %v, got %v", i, path, e, a))
 					}
-					if d > 15*time.Second {
-						recordError(fmt.Sprintf("%v: path %v took %v > 15s", i, path, d))
-					}
 				}(i, path, val)
 				time.Sleep(150 * time.Millisecond)
```

Comment:
@liggitt makes sense, thanks for the debugging.

I still think that testing duration here doesn't make sense.  If we want to validate SLA we should write a different test.